differ/differDriver.go

```diff
@@ -391,7 +401,15 @@ func (dh *DifferHandler) run() error {
 				}
 				dh.writeDiffBytes(diffBytes)
 			}
+			srcVbItemCnt += filesDiffer.file1ItemCount
```

Comment:
This probably needs updating after the prior comment is addressed.

Reply:
Updated